test/pkg/integration/setup.go

```diff
@@ -159,34 +164,48 @@ func waitOnGitpodRunning(namespace string, waitTimeout time.Duration) env.Func {
 
 		client := cfg.Client()
 		err := wait.PollImmediate(1*time.Second, waitTimeout, func() (bool, error) {
+			var wg sync.WaitGroup
+			ready := true
 			for _, component := range components {
```

Comment:
This check was taking 4-5s and runs before each component to test, parallelizing the component checks reduces it to <1s